docs/configuration.md

```diff
@@ -392,6 +392,16 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td>Int.MaxValue</td>
+  <td>
+    "spark.reducer.maxMbInFlight" puts a bound on the in flight data in terms of size.
+    But this is not always sufficient when the number of hosts in the cluster increase,
```

Comment:
maxMBInFlight should be maxSizeInFlight

change "to fetches" to "to fetch"

Wrap the actual config variables with 

I would prefer to see the description of spark.reducer.maxReqsInFlight first. Perhaps after that put something close to what you have about the maxMbInFlight. something like (feel free to change wording).

maxReqsInFlight limits number ..... This is sometimes needed when you have a large number of hosts and which can lead to a very large number of in-bound connections...